packages/client/lib/Chain.js

```diff
@@ -226,4 +234,16 @@ export default class Chain {
   async getAddressMempool (addresses) {
```

Comment:
Can you explain why `getAddressMempool`, `getCode` `erc20Approve`, `getTransactionReceipt` are needed on client.chain object?